[2024-09-20T23:20:27.174+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-09-20T23:20:27.366+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_dag_v45.insert_orders_data manual__2024-09-20T23:11:48.499967+00:00 [queued]>
[2024-09-20T23:20:27.397+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_dag_v45.insert_orders_data manual__2024-09-20T23:11:48.499967+00:00 [queued]>
[2024-09-20T23:20:27.399+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 6
[2024-09-20T23:20:27.481+0000] {taskinstance.py:2888} INFO - Executing <Task(PostgresOperator): insert_orders_data> on 2024-09-20 23:11:48.499967+00:00
[2024-09-20T23:20:27.497+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'etl_dag_v45', 'insert_orders_data', 'manual__2024-09-20T23:11:48.499967+00:00', '--job-id', '1632', '--raw', '--subdir', 'DAGS_FOLDER/etl_dag.py', '--cfg-path', '/tmp/tmpunkpwsyn']
[2024-09-20T23:20:27.504+0000] {standard_task_runner.py:105} INFO - Job 1632: Subtask insert_orders_data
[2024-09-20T23:20:27.513+0000] {logging_mixin.py:190} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=6156) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-09-20T23:20:27.515+0000] {standard_task_runner.py:72} INFO - Started process 6161 to run task
[2024-09-20T23:20:27.736+0000] {task_command.py:467} INFO - Running <TaskInstance: etl_dag_v45.insert_orders_data manual__2024-09-20T23:11:48.499967+00:00 [running]> on host 217a74bc1b7d
[2024-09-20T23:20:28.474+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='etl_dag_v45' AIRFLOW_CTX_TASK_ID='insert_orders_data' AIRFLOW_CTX_EXECUTION_DATE='2024-09-20T23:11:48.499967+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-09-20T23:11:48.499967+00:00'
[2024-09-20T23:20:28.477+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-09-20T23:20:28.558+0000] {sql.py:266} INFO - Executing: 
            TRUNCATE TABLE orders;
            COPY orders(id, customer_id, product_id, order_date, quantity)
            FROM '/shared/orders.csv'
            DELIMITER ',' CSV HEADER;
        
[2024-09-20T23:20:28.798+0000] {base.py:84} INFO - Retrieving connection 'postgres_localhost'
[2024-09-20T23:20:29.267+0000] {base.py:84} INFO - Retrieving connection 'postgres_localhost'
[2024-09-20T23:20:29.391+0000] {sql.py:509} INFO - Running statement: 
            TRUNCATE TABLE orders;
            COPY orders(id, customer_id, product_id, order_date, quantity)
            FROM '/shared/orders.csv'
            DELIMITER ',' CSV HEADER;
        , parameters: None
[2024-09-20T23:20:29.424+0000] {sql.py:518} INFO - Rows affected: 50
[2024-09-20T23:20:29.494+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-09-20T23:20:29.495+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=etl_dag_v45, task_id=insert_orders_data, run_id=manual__2024-09-20T23:11:48.499967+00:00, execution_date=20240920T231148, start_date=20240920T232027, end_date=20240920T232029
[2024-09-20T23:20:29.744+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-09-20T23:20:29.814+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-09-20T23:20:29.821+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
